Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/ActionList.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Maestro.WebApi.Models
+{
+    public class ActionsList
+    {
+        [JsonExtensionData]
+        private IDictionary<string, JToken> ActionNames { get; set; }
```

Comment:
Ah, you're right (and I see now you just moved this class, so it's fine anyway). I don't see properties mentioned in the doc, but it's the default way. I guess I'm just not used to seeing private properties.